distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -257,14 +257,19 @@ karaf.default.repository=system
 # additional commands.
 # Do not use absolute paths to avoid problems on windows.
 #
-karaf.shell.init.script=etc/shell.init.script,etc/scripts/*.script
+karaf.shell.init.script=shell.init.script,scripts/*.script
 
 #
 # Sets the maximum size of the shell command history. If not set,
 # defaults to 500 entries. Setting to 0 will disable history.
 #
 # karaf.shell.history.maxSize = 0
```

Comment:
✏️ Support was added in Karaf 4.2.0 for overriding values in `system.properties` (https://github.com/apache/karaf/commit/7fb0a652b27ad15f2106d985681cba0e76b5f75b). Do we want to switch to use `custom.system.properties` instead to keep this file more maintainable?

Reply:
✏️ `karaf.shell.history.maxSize` and `karaf.shell.history.file.maxSize` are commented out in custom.system.properties and can be removed too, right?

✏️ The `#START DDF SETTINGS` comment can be removed.

✏️ The Apache header should be removed at the beginning of this file since it should only contain custom DDF stuff, right?